deployments/properties/aks.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to a AKS Cluster",
```

Comment:
Should it be named to "Deploy to a AKS Cluster using Azure Credentials", to distinguish it from template added in https://github.com/actions/starter-workflows/pull/1206

Reply:
Deploy to "AKS cluster" seems to distinguish it pretty clearly?